content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -315,6 +316,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `AdvancedAuditing`: Enable [advanced auditing](/docs/tasks/debug-application-cluster/audit/#advanced-audit)
 - `AffinityInAnnotations`(*deprecated*): Enable setting [Pod affinity or anti-affinity](/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity).
 - `AllowExtTrafficLocalEndpoints`: Enable a service to route external requests to node local endpoints.
+- `AnyVolumeDataSource`: Enable use of any custom resource as the `DataSource` of a `PersistentVolumeClaim`.
```

Comment:
nit; I'd write this using a [glossary](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#glossary) tooltip (especially as it's so high up in the alphabet):
```suggestion
- `AnyVolumeDataSource`: Enable use of any custom resource as the `DataSource` of a 
  {{< glossary_tooltip text="PVC" term_id="persistent-volume-claim" >}}.
```